cv32e40p/sim/core/Makefile

```diff
@@ -53,6 +53,19 @@ CV_CORE       ?= CV32E40P
 CV_CORE_LC     = $(shell echo $(CV_CORE) | tr A-Z a-z)
 CV_CORE_UC     = $(shell echo $(CV_CORE) | tr a-z A-Z)
 
+SIMULATOR     ?= $(CV_SIMULATOR)
+
+# Source directores
+VERI_CUSTOM = $(CORE_V_VERIF)/$(CV_CORE_LC)/tests/programs/custom
+
+# Common output directories
+RUN_INDEX               ?= 0
+SIM_RESULTS              = simulation_results
+SIM_TEST_RESULTS         = $(SIM_RESULTS)/$(TEST)
+SIM_RUN_RESULTS          = $(SIM_TEST_RESULTS)/$(RUN_INDEX)
+SIM_TEST_PROGRAM_RESULTS = $(SIM_RUN_RESULTS)/test_program
+SIM_BSP_RESULTS          = $(SIM_TEST_PROGRAM_RESULTS)/bsp
```

Comment:
This are now done the same way as is done in "mk/uvmt/uvmt.mk"? That seems nice, to do it all in the same consistent way.

I don't know if maybe it could make sense to extract these common "paths-related" settings into one makefile fragment that all other makefiles could include?
Also I see that the "sim/uvmt/Makefile" just includes "mk/uvmt/uvmt.mk", but the "sim/core/Makefile" does not just include "mk/core/core.mk" (but if it works it works).

Reply:
It is _almost_ the same.